src/main/resources/application.yml

```diff
@@ -29,10 +32,6 @@ spring:
     hikari:
       maximum-pool-size: 25
 
-  cache:
-    cache-names: userDetails
-    caffeine.spec: maximumSize=2000,expireAfterWrite=300s,recordStats
-
```

Comment:
How come you made this hardcoded in the config class?

Reply:
because I didn't want generic config for all caches, I wanted specific config for each cache separately